.golangci.yml

```diff
@@ -54,11 +54,10 @@ issues:
   max-same-issues: 10000
 
 linters-settings:
-  dogsled:
-    max-blank-identifiers: 3
-  maligned:
-    # print struct with more effective memory layout or not, false by default
-    suggest-new: true
+  revive:
+    # When set to false, ignores files with "GENERATED" header, similar to golint
```

Comment:
This comment is very confusing for me. Shouldn't it be `When set to true` ? I saw this in the original doc as well. 🤷